frontend/src/containers/LoginPage/LoginPage.js

```diff
@@ -19,10 +20,10 @@ class LoginPage extends React.Component {
     authenticate(mail, password)
       .then((responseToken) => {
         const token = responseToken.data;
-        getCurrentUser(token)
+        window.sessionStorage.setItem('token', token);
```

Comment:
I don't think moving this is a good idea. If the first request goes through and the second one does not (unlikely scenario) user would have a JWT token that would redirect him to Library page. However, we would not have all of his information. I think that we should save things to sessionStorage only when all of the requests go through successfully.